Assets/MRTK/StandardAssets/Shaders/MixedRealityTextMeshProSprite.shader

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// NOTE: MRTK Shaders are versioned via the MRTK.Shaders.sentinel file.
+// When making changes to any shader's source file, the value in the sentinel _must_ be incremented.
+
+// Text Mesh Pro Sprite shader with MRTK Additions
+
+// MRTK Additions
+// - Single Pass Instanced Stereo Rendering Support
+// - Support for Clipping Primitives (Plane, Sphere, Box)
+// - Added to MRTK namespace
+
+Shader "Mixed Reality Toolkit/TextMeshProSprite"
+{
+	Properties
+	{
+		_MainTex ("Sprite Texture", 2D) = "white" {}
+		_Color ("Tint", Color) = (1,1,1,1)
+		
+		_StencilComp ("Stencil Comparison", Float) = 8
+		_Stencil ("Stencil ID", Float) = 0
+		_StencilOp ("Stencil Operation", Float) = 0
+		_StencilWriteMask ("Stencil Write Mask", Float) = 255
+		_StencilReadMask ("Stencil Read Mask", Float) = 255
+
+		_ColorMask ("Color Mask", Float) = 15
+		_ClipRect ("Clip Rect", vector) = (-32767, -32767, 32767, 32767)
+
+		[Toggle(UNITY_UI_ALPHACLIP)] _UseUIAlphaClip ("Use Alpha Clip", Float) = 0
+	}
+
+	SubShader
+	{
+		Tags
+		{ 
+			"Queue"="Transparent" 
+			"IgnoreProjector"="True" 
+			"RenderType"="Transparent" 
+			"PreviewType"="Plane"
+			"CanUseSpriteAtlas"="True"
+		}
+		
+		Stencil
+		{
+			Ref [_Stencil]
+			Comp [_StencilComp]
+			Pass [_StencilOp] 
+			ReadMask [_StencilReadMask]
+			WriteMask [_StencilWriteMask]
+		}
+
+		Cull Off
+		Lighting Off
+		ZWrite Off
+		ZTest [unity_GUIZTestMode]
+		Blend SrcAlpha OneMinusSrcAlpha
+		ColorMask [_ColorMask]
+
+		Pass
+		{
+		CGPROGRAM
+			#pragma vertex vert

```

Comment:
It looks like the formatting of this file got a little wonky (maybe tabs vs spaces?). If Visual Studio can't [auto format](https://stackoverflow.com/questions/5755942/how-do-you-auto-format-code-in-visual-studio) the file you can turn on [whitespace viewing](https://stackoverflow.com/questions/4065815/how-to-turn-off-showing-whitespace-characters-in-visual-studio-ide) and do it manually.

Reply:
I could not get Visual Studio to auto format this shader, so I went and changed it manually. The changes have been committed to this PR!